batch/sql/add-seqr-nonpreemptible-pools.sql

```diff
@@ -0,0 +1,44 @@
+-- Adds dedicated pools for the seqr loading pipeline, with the 'seqr' pool label, but non-preemptible versions 
```

Comment:
You could potentially add these changes simply to `add-seqr-pools.sql`, as we haven't integrated that with CI anyhow.

I.e. regarding your question of reproducibility: ideally we'd add all our "CPG"-custom pools in here (including the `large-cohorts` one, so we should probably rename this to `add-cpg-pools.sql` or similar) and add another build step in `build.yaml`, so these pools also get created for dev deployments, for example. I was hoping to do that once the upstream merge had been done, but that ended up taking way longer (in fact has just been recently merged!).